### PR TITLE
Refactor to RV3

### PR DESCRIPTION
**Pull request by tiny-tinker**

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -1,306 +1,406 @@
...
+  import logs from '../assets/img/logs.png';
+  import slack from '../assets/img/slack.png';
+  import hoverPoint from '../assets/img/hover-point.png';
+  import storyscript from '../assets/img/storyscript.png';
+  import storyscriptLeft from '../assets/img/storyscript-left.png';
+  import storyscriptRight from '../assets/img/storyscript-right.png';
+  import dashboard from '../assets/img/dashboard.png';
+  import hub from '../assets/img/hub.png';
+  import benefitPlaceholder from '../assets/img/benefit-placeholder.svg';
+  import nebula from '../assets/img/nebula.png';
```
Comment (JeanBarriere) [inline pages/index.vue]: why importing all the assets here and not directly with `@/assets/something.png` in the template ?

File: components/Header/Header.vue [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+											</a>
+			</li>
+			<li class="nav-item">
+					<a class="nav-link nav-link-icon" href="#">
+													Events
+											</a>
+			</li>
+			<li class="nav-item">
+					<a class="nav-link nav-link-icon" href="#">
+													Contact
```
[the remark this answers is not in the document]
Comment (tiny-tinker) [reply components/Header/Header.vue]: Ops! That's what "beautify vue" command does. I will fix it

File: pages/index.vue (hunk 1 above)
Comment (tiny-tinker) [reply pages/index.vue]: I wanted to put all values in one place. Anyway used direct URL as you mentioned. 👍

**Comment by stevepeak** [comment]
Moved to https://github.com/asyncy/asyncy.com-next

**Pull request closed without merging**